src/main/java/io/cdap/plugin/gcp/bigquery/relational/SQLExpression.java

```diff
@@ -0,0 +1,69 @@
+package io.cdap.plugin.gcp.bigquery.relational;
+
+import io.cdap.cdap.etl.api.relational.Expression;
+
+import java.util.Objects;
+
+/**
+ * A default implementation of Expression that simply stores the expression as a string.
+ * The string is assumed to be ANSI SQL compliant.
+ * It is the responsibility of the factory creating {@link SQLExpression} objects to enforce correctness of
+ * the expression.
+ */
+public class SQLExpression implements Expression {
+  private final String expression;
+
+  /**
+   * Creates a {@link SQLExpression} from the specified SQL string.
+   * @param expression a String containing the SQL expression.
+   */
+  public SQLExpression(String expression) {
+    this.expression = expression;
+  }
+
+  public String getExpression() {
```

Comment:
nit: add a javadoc here saying "This returns the wrapped expression" or something like this.

Reply:
Done.